src/components/sidenav.jsx

```diff
@@ -0,0 +1,100 @@
+import { NavLink } from "react-router-dom";
+
+const sidebarItems = [
+  {
+    name: "Avatars",
+  },
+  {
+    name: "Alert",
+  },
+  {
+    name: "Badges",
+  },
+  {
+    name: "Buttons",
+  },
+  {
+    name: "Cards",
+  },
+  {
+    name: "Image",
+  },
+  {
+    name: "Input",
+  },
+  {
+    name: "Lists",
+  },
+  {
+    name: "Modal",
+  },
+  {
+    name: "Typograhy",
+  },
+  {
+    name: "Toasts",
+  },
+];
+
+export const Sidenav = ({ showSidenav }) => {
+  return (
+    <div
+      className="sidenav"
+      style={
+        showSidenav
+          ? {
+              display: "none",
+              position: "absolute",
+              zIndex: "3",
+              transition: "0.4s",
+              width: "100vw",
+              height: "100vh",
+            }
+          : { display: "flex" }
```

Comment:
we can avoid the inline style here by toggling the className instead.
```suggestion
      <div className={sideNav ? "sidenav styles1" : "sidenav styles2"> </div>
      
```

Reply:
I decided to use inline CSS because it has higher preference. With using `classname` the app breaks.